Add trigger and condition to configuration vars (#22467)
home-assistant/home-assistant.github.io: source/_docs/automation/yaml.markdown

  required: false
  type: string
  default: warning
trigger:

>   description: The trigger(s) which will start the automation. Multiple triggers can be added and the automation will start when any of these triggers changes from `false` to `true`.

Reviewer: Triggers are not conditional (e.g., can be event based too); so its not about being true or false.

```suggestion
  description: The trigger(s) which will start the automation. Multiple triggers can be added and the automation will start when any of these triggers trigger.
```

  required: true
  type: list
  keys:
    id:

>       description: An id which can be used in the automatin to determine which trigger caused the automation to start

Reviewer: ```suggestion
      description: An ID that can be used in the automation to determine which trigger caused the automation to start.
```

      type: string
      required: false
    variables:

>       description: Variables which will be available in the conditions and action sequence.

Reviewer: ```suggestion
      description: Variables that will be available in the conditions and action sequence.
```

      required: false
      default: {}
      type: map
      keys:
        PARAMETER_NAME:
          description: The value of the variable. Any YAML is valid. Templates can also be used to pass a value to the variable.
          type: any
condition:

>   description: Conditions which have to be `true` to start the automation. By default all conditions listed have to be `true`, you can use [logical conditions](https://www.home-assistant.io/docs/scripts/conditions/#logical-conditions) to change this default behaviour.

Reviewer: ```suggestion
  description: Conditions that have to be `true` to start the automation. By default all conditions listed have to be `true`, you can use [logical conditions](/docs/scripts/conditions/#logical-conditions) to change this default behavior.
```
